### PR TITLE
N16: Add Antorum Online

### PR DESCRIPTION
**Pull request by Dooskington**
Part of #365 ("Newsletter 16")

### CODE REVIEW

File: content/posts/newsletter-016/index.md [hunk 1]
```diff
@@ -63,6 +63,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Antorum Online]
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-016/index.md:66]: If you want this to link somewhere, you need a corresponding link at the bottom of the post (like you've done for your dev feed and Twitter handle). If you just want it to be a normal heading, you can take off the `[]` around it :)

File: content/posts/newsletter-016/index.md [hunk 2]
```diff
@@ -63,6 +63,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+Many important changes and new features were implemented this month. Players can
+now harvest plants and go fishing! There were also some tweaks to the world
+engine to support named zones on the map.
+
+- [23: Herbology](https://ratwizard.dev/dev-log/antorum/23)
+- [24: Fishing](https://ratwizard.dev/dev-log/antorum/24)
+- [25: Zones](https://ratwizard.dev/dev-log/antorum/25)
+
+[Antorum Online Development Feed]: https://ratwizard.dev/dev-log/antorum
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-016/index.md:82]: You'd need `[Antorum Online Development Feed]` somewhere in the text of the post for this link to be displayed. Can you either add a link or remove this line?

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
Content looks good, just a couple of little link changes that need making:

File: content/posts/newsletter-016/index.md (hunk 1 above)
Comment (Dooskington) [reply content/posts/newsletter-016/index.md:66]: Shoot! I made the same mistake again. Will fix

**Review by 17cupsofcoffee — APPROVED** [review]
Looks good :+1:

**Review by ozkriff — COMMENTED** [review]
:+1:

**Comment by ozkriff** [comment]
Merged, thanks!

**Pull request merged**